Johnny-jbuccola/homework_11/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Advanced APIs Lesson</title>
+
+  <!-- bootstrap -->
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
+  <!-- css -->
+  <link rel="stylesheet" href="/styles.css">
+  <!-- jQuery -->
+  <script src="https://code.jquery.com/jquery-2.2.3.min.js"></script>
+</head>
+<body>
+
+  <header class="container-fluid text-center">
+    <h1>Local<span>Landscapes</span></h1>
+  </header>
+
+  <div class="container-fluid sign-in-view">
+    <div class="row text-center">
+      <h3>Discover how beautiful your surroundings can be</h3>
+      <button class="btn btn-default">
+        <a href="#" id="login">Login to 500px</a>
+      </button>
```

Comment:
Consideration for later when we work with auth functionality: 
When a user is logged in, what happens to this `.login` id? Would you use templates to hide/show a login button depending on whether the user is authenticated? Would you use vanilla JS/jQuery to build the page upon determining authentication status?